src/test/java/org/jvnet/hudson/test/JenkinsRuleTest.java

```diff
@@ -92,4 +182,27 @@ public String getSetterParam() {
             return setterParam;
         }
     }
+
+    @TestExtension
```

Comment:
Maybe it should be linked to a particular test case. "WhoAmI" name seems to be copy-pasta

Reply:
Name changed + link added to the new test